app/web/src/features/communities/events/EventForm.tsx

```diff
@@ -145,7 +155,10 @@ export default function EventForm({
 
   const onSubmit = handleSubmit(
     (data) => {
-      mutate(data);
+      mutate({
+        ...data,
+        dirtyFields,
+      });
```

Comment:
why not send the dirty fields only?

Reply:
Initially I was thinking that'd cause problems since the fields in that object is dynamic, but actually it'd just return `undefined` so I guess no good reasons really, could do that too.

Reply:
Wait actually that's not true, pretty much all the fields in `dirtyFields` are used to determine what to send to the backend, so it's just easier to pass the whole object through rather than field by field.

Reply:
I mean `mutate({ ...dirtyfields })`

Reply:
That wouldn't work since `dirtyFields` don't give the actual value of the field, it's only an object of `true`s for fields that's dirty (e.g. if `title` is dirty, `dirtyFields` = `{ title: true }`.